xml/article_sap_monitoring.xml

```diff
@@ -110,72 +110,480 @@ https://suse.com/c/coming-soon-sap-landscape-monitoring-on-sles-for-sap/
+          certification validation. This option is required by &hana;
+          cloud. Only available for the <literal>dbapi</literal> connector.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          <literal>hana.aws_secret_name</literal> Secret name containing the
+          username and password (see <xref linkend="sec-sol-monit-hana-db-aws-secret"/>. Use this option when &hana; database is
+          stored on AWS. <literal>aws_secret_name</literal> and
+          <literal>user/password</literal> are mutually exclusive. If both are
+          set, <literal>aws_secret_name</literal> takes priority.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          <literal>logging.config_file</literal> Python logging
+          system configuration file (by default WARN and ERROR level
+          messages are sent to the syslog).
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          <literal>logging.log_file</literal>: Logging file
+          (<filename>/var/log/hanadb_exporter.log</filename> by default)
+        </para>
+      </listitem>
+    </itemizedlist>
+    <para>
+      The logging configuration file follows the
+      <link
+      xlink:href="https://docs.python.org/3/library/logging.config.html">Python standard
+      logging system style</link>.
+    </para>
+    <para>
+      Using the default
+      <link xlink:href="./logging_config.ini">configuration
+      file</link>, it will redirect the logs to the file assigned in
+      the <link xlink:href="./config.json.example">json configuration
+      file</link> and to the syslog (only logging level up to
+      WARNING).
+    </para>
+    </sect2>
+    <sect2 xml:id="sec-sol-monit-hana-db-stored-key">
+      <title>Using the stored user key</title>
+      <para>
+        Use this option if we want to keep the database
+        secure (you can use
+        <literal>user/password</literal> with the
+        <literal>SYSTEM</literal> user for development, as it is faster to set
+        up). To use the <literal>userkey</literal> option, the
+        <literal>dbapi</literal> must be installed (normally stored in
+        <filename>/hana/shared/<replaceable>SID</replaceable>/hdbclient/hdbcli-N.N.N.tar.gz</filename>
+        and installable with <package>pip3</package>). The key is stored in the client itself. To use a different client, you must create a new stored user key for the user running Python
+        
+        It cannot be used with a
+        different client, as the key is stored in the client itself.
+        This raises the
+        <literal>hdbcli.dbapi.Error: (-10104, 'Invalid value for KEY')</literal>
+        error. For that a new stored user key must be created with the
+        user that is running python. For that (please, notice that the
+        <literal>hdbclient</literal> is the same as the
+        <literal>dbapi</literal> python package):
```

Comment:
This also looks like a repetition of a preceding sentence. Can we replace the second "For that" with a more detailed instruction?